samples/code-samples/DatabaseManagement/Program.cs

```diff
@@ -51,10 +51,19 @@ public static void Main(string[] args)
                     RunDatabaseDemo().Wait();
                 }
             }
-            catch (DocumentClientException de)
+            catch (AggregateException age)
```

Comment:
As discussed offline, move this try catch in the below method where you are actually calling Cosmos DB APIs. That way we can keep it clean and also catch DocumentClientException(specific exception).